src/components/MayHaveMissed.vue

```diff
@@ -10,7 +10,10 @@
   import { ref, onMounted } from 'vue';
   import { getRandomNumbersInRange } from './helpers/sprinkle_getRandomNumbersInRange'
   
-  const articleNumber: any[] = [getRandomNumbersInRange(1, 0, 24)]; //eventually get this thing to pick
+  // Define props for the component
+  const { articleNumber = 4 } = defineProps<{
+    articleNumber: number
+  }>(); //eventually get this thing to pick
```

Comment:
I imagine this killed some errors, but haven't looked in-browser yet.

If so, nice.